lineapy/graph_reader/types.py

```diff
@@ -6,13 +6,18 @@ class InputVariable:
     """
     Class to generate code related input variable and it's default value
 
-    :param variable_name: variable name
-    :param value: variable value
-    :param value_type: variable objec type
+    Parameters
```

Comment:
Rename to `Attributes`

Reply:
I redid the whole block of comments here to improve it.